### PR TITLE
fix: Remove old @microsoft/recognizers-text-number version with postinstall scripts

### PR DESCRIPTION
**Pull request by JhontSouth**
#minor

## Description
This PR removes the old version or **_@microsoft/recognizers-text-number_** from the other recognizers packages that require it.

## Specific Changes
  - Added postinstall scripts in botbuilder-dialogs and botbuilder-dialogs-adaptive package.json files to remove the old version of  **_@microsoft/recognizers-text-number_**.

## Testing
These images show the dependencies tree before and after running the postinstall scripts.
image

### CODE REVIEW

File: libraries/botbuilder-dialogs-adaptive/package.json [hunk 1]
```diff
@@ -47,7 +47,8 @@
     "node-fetch": "^2.6.7"
   },
   "devDependencies": {
-    "@types/node-fetch": "^2.5.3"
+    "@types/node-fetch": "^2.5.3",
+    "rimraf": "^3.0.2"
```
Comment (ceciliaavila) [inline libraries/botbuilder-dialogs-adaptive/package.json:51]: Is it necessary to include rimraf if you are using npx to run the command?

File: libraries/botbuilder-dialogs/package.json [hunk 2]
```diff
@@ -41,7 +41,8 @@
   },
   "devDependencies": {
     "@types/globalize": "^1.5.0",
-    "line-reader": "^0.4.0"
+    "line-reader": "^0.4.0",
+    "rimraf": "^3.0.2"
```
Comment (ceciliaavila) [inline libraries/botbuilder-dialogs/package.json:45]: Same here.

**Review by ceciliaavila — CHANGES_REQUESTED** [review]

File: libraries/botbuilder-dialogs-adaptive/package.json (hunk 1 above)
Comment (JhontSouth) [reply libraries/botbuilder-dialogs-adaptive/package.json:51]: Without the _rimraf_ it returns an error that the reference was not found.

**Review by sw-joelmut — APPROVED** [review]

**Comment by JhontSouth** [comment]
Promoted /4608

**Pull request closed without merging**